### PR TITLE
SONAR-7986 Accept multiple keys in api/settings/rest

### PR DESCRIPTION
**Pull request by julienlancelot**
SONAR-7986 Accept multiple keys in api/settings/rest

### CODE REVIEW

File: it/it-tests/src/test/java/it/settings/PropertySetsTest.java [hunk 1]
```diff
@@ -129,7 +129,7 @@ private void assertPropertySet(String baseSettingKey, List<Map.Entry<String, Str
     assertThat(setting.getFieldValues().getFieldValuesList()).hasSize(fieldsValues.length);
     int index = 0;
     for (Settings.FieldValues.Value fieldsValue : setting.getFieldValues().getFieldValuesList()) {
```
Comment (teryk) [inline it/it-tests/src/test/java/it/settings/PropertySetsTest.java]: minor: rename to fieldValues

File: it/it-tests/src/test/java/it/settings/SettingsTest.java [hunk 2]
```diff
@@ -87,24 +88,28 @@ public void get_default_value() throws Exception {
...
+  public void set_setting() throws Exception {
     SETTINGS.set(SetRequest.builder().setKey(PLUGIN_SETTING_KEY).setValue("some value").build());
 
     String value = getSetting(PLUGIN_SETTING_KEY).getValue();
     assertThat(value).isEqualTo("some value");
   }
 
   @Test
-  public void remove_value() throws Exception {
+  public void remove_setting() throws Exception {
```
Comment (teryk) [inline it/it-tests/src/test/java/it/settings/SettingsTest.java]: can you update this IT to remove several settings?

File: server/sonar-server/src/main/java/org/sonar/server/setting/ws/SettingsUpdater.java [hunk 3]
```diff
@@ -42,49 +45,63 @@ public SettingsUpdater(DbClient dbClient, PropertyDefinitions definitions) {
...
-    delete(dbSession, propertyKey, Optional.empty());
+  public void deleteGlobalSettings(DbSession dbSession, String... settingKeys) {
+    deleteGlobalSettings(dbSession, asList(settingKeys));
   }
 
-  public void deleteComponentSetting(DbSession dbSession, String propertyKey, ComponentDto componentDto) {
-    delete(dbSession, propertyKey, Optional.of(componentDto));
+  public void deleteGlobalSettings(DbSession dbSession, List<String> settingKeys) {
+    checkArgument(!settingKeys.isEmpty(), "At least one setting key is required");
+    for (String propertyKey : settingKeys) {
```
Comment (teryk) [inline server/sonar-server/src/main/java/org/sonar/server/setting/ws/SettingsUpdater.java]: remark: you can use settingKeys.forEach, and call is settingKey or key but not propertyKey

**Comment by teryk** [comment]
if you confirm that you can't have a property with a comma, apply feedbacks and LGTM

File: it/it-tests/src/test/java/it/settings/SettingsTest.java (hunk 2 above)
Comment (julienlancelot) [inline it/it-tests/src/test/java/it/settings/SettingsTest.java]: Can you please have a better look of what I've done :) ?

Comment (teryk) [inline it/it-tests/src/test/java/it/settings/SettingsTest.java]: 😭 sorry!

**Pull request merged**